REDCapUITweaker.php

```diff
@@ -3641,18 +3673,30 @@ function sqlCheckboxAddSQL()
 	{
 		unset( $_POST['dropdown_autocomplete'] );
 		$_POST['element_enum'] =
-			"SELECT * FROM ( WITH RECURSIVE redcap_ui_tweaker_qy AS (\n" .
+			"SELECT * FROM ( WITH RECURSIVE redcap_ui_tweaker_ev AS ( SELECT DISTINCT `value` " .
+			"AS v FROM [data-table] WHERE project_id = [project-id] AND field_name = '" .
+			preg_replace( '/[^A-Za-z0-9_]/', '', $_POST['field_name'] ) .
+			"' UNION SELECT v FROM redcap_external_module_settings JOIN JSON_TABLE( `value`, " .
+			"'$[*]' COLUMNS ( v VARCHAR(255) PATH '$' ) ) redcap_ui_tweaker_sv WHERE " .
+			"external_module_id = (SELECT external_module_id FROM redcap_external_modules WHERE " .
+			"directory_prefix = 'redcap_ui_tweaker' LIMIT 1) AND project_id = [project-id] AND " .
+			"`key` = 'sql-checkbox-submittedvalues' ), redcap_ui_tweaker_qy AS (" .
+			"SELECT * FROM (\n" .
 			"-- END redcap_ui_tweaker\n" .
 			rtrim( $_POST['element_enum'], " \n\r\t;" ) .
 			"\n-- BEGIN redcap_ui_tweaker\n" .
+			") redcap_ui_tweaker_sqy LIMIT 60" .
```

Comment:
This limit is rather buried in the code, and is a numeric literal.  I would prefer a constant to be used.